test/test/spawn-bail.tap

```diff
@@ -15,8 +15,7 @@ TAP version 13
               ---
               {"found":1,"wanted":"1","compare":"===","at":{"file":"test/test/spawn.js","line":22,"column":10,"function":"foo"},"source":"tt.equal(1, '1', 'nested failure')\n"}
               ...
-            Bail out! # nested failure
-        Bail out! # nested failure
+
```

Comment:
When I re-generate the fixture, it puts it back where it is expected to be... so... yeah, there's definitely still some kind of race there.